providers/yttcb/clusterbootstrap.yaml

```diff
@@ -160,7 +160,7 @@ stringData:
 #@ end
 #@ if antrea_config_customized():
 ---
-apiVersion: cni.tanzu.vmware.com/v1alpha1
+apiVersion: cni.tanzu.vmware.com/v1alpha2
```

Comment:
@shyaamsn this file will be used only when the ANTREA variable is passed... what about the template in the paravirt package, does it needs to be bumped as well, what is the impact?

https://github.com/vmware-tanzu/tanzu-framework/blob/52d730db4a43f6ffabdf27ae80f74bfce0062999/addons/config/templates/cni.tanzu.vmware.com/v1alpha1/antreaconfig.yaml#L3